docs/tutorials/comet-impact.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
I think this is a nice introduction to 'scipp as a calculator' as you put it. But I have 2 main comments:
- There isn't really any motivation for using arrays. It would be better to ultimately show a plot of, e.g., the temperature change vs velocity, or mass of the comet.
- This tutorial takes a science centered viewpoint. This is not bad in itself but it might make for a bad starting point for people with a data science background.

Reply:
>  It would be better to ultimately show a plot of, e.g., the temperature change vs velocity, or mass of the comet.

I agree, we would really like to use data arrays for this. So should we skip the array bit entirely here?

Reply:
> This tutorial takes a science centered viewpoint. This is not bad in itself but it might make for a bad starting point for people with a data science background.

Which aspect? The fact that we use units?

Reply:
> > It would be better to ultimately show a plot of, e.g., the temperature change vs velocity, or mass of the comet.
> 
> I agree, we would really like to use data arrays for this. So should we skip the array bit entirely here?

Could also make an ad hoc data array with a comment that it is explained elsewhere. Or use matplotlib directly.
I think showing arrays is valuable because it shows that the other stuff is not limited to scalars.


> > This tutorial takes a science centered viewpoint. This is not bad in itself but it might make for a bad starting point for people with a data science background.
> 
> Which aspect? The fact that we use units?

No, units are great, But it calculated some quantities based on physical equations and doesn't get / load data.